lib/livebook_web/live/session_live.ex

```diff
@@ -137,7 +137,7 @@ defmodule LivebookWeb.SessionLive do
       </div>
       <div class="grow overflow-y-auto relative" data-el-notebook>
         <div data-el-js-view-iframes phx-update="ignore" id="js-view-iframes"></div>
-        <div class="w-full max-w-screen-lg px-16 mx-auto py-7" data-el-notebook-content>
+        <div class="w-full max-w-screen-lg pl-8 pr-16 mx-auto py-7" data-el-notebook-content>
```

Comment:
For larger screens I'd keep it centered, wdyt?

```suggestion
        <div class="w-full max-w-screen-lg pl-8 md:pl-16 pr-16 mx-auto py-7" data-el-notebook-content>
```

Reply:
Sounds good. Pls merge and ship!